multimodal/package_description.md

```diff
@@ -0,0 +1,28 @@
+# Microsoft Health Intelligence Multimodal Toolbox
+
+## Overview
+
+This toolbox provides models for working with multi-modal health data.
+
+## Installation
+
+You can install the latest version from `pypi` via
+
+```shell
+pip install hi-ml-multimodal
+```
+
+## Documentation
+
+The detailed package documentation, with examples and API reference, is on
+[readthedocs](https://hi-ml.readthedocs.io/en/latest/).
```

Comment:
Should we point to project specific links on Read The Docs? I see one for hi-ml-histopathology (https://hi-ml.readthedocs.io/en/latest/histopathology.html) but not for hi-ml.

Also, not too important now, but how do we want to point to older versions of our docs, given that each package will move independently? Maybe we can come up with some nomenclature to make it clear what package each release/version based on.

Reply:
Yes, this should point to something specific - but for now, these are all just templates that need to be filled out later once we have actual docs.